_component_design/date-picker.md

```diff
@@ -0,0 +1,341 @@
+---
+title: Date Picker
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: Enable users to filter a list of events, or a report, by date ranges, or choose a single date for an event as they fill out a form.
+preview-image: preview-images/date-picker.svg
+status: unknown
+resource: true
+last-modified: 2017-10-22
+---
+
+{% include toc.html %}
+
+## Introduction
+
```

Comment:
Add `{: .hxSectionTitle}`

Reply:
Leaving a note for memory: We're taking these class name calls out per @araiford: https://github.com/rackerlabs/design-system/pull/292#issuecomment-338716258